### PR TITLE
Feature/grid display styling ect

### PR DESCRIPTION
**Pull request by ectweitmann**
## What Features are Being Added?
- Created class styles for stylizing both the landing page and favorites page. 
- Added classNames to JSX elements

## What is Being Refactored?
- Minor tweak removing button element from BookCard component because it wasn't necessary.

## Where should the reviewer start?
- App, BookContainer, BookCard, and Nav components/.css files

## How should this be manually tested?
- npm start and investigate that the landing page renders correctly in a 3 x 30 grid

## What are the relevant tickets?
- Landing/favorites page styling

## Any Other Notes/Questions/Problems?
- The styles are not responsive yet so the page will break when resized to smaller screen sizes. 
- Also need to update one of the books cover images in the data to be inline with the rest of cover images. It's currently a square we need it to be a rectangle.

### CODE REVIEW

File: src/styles/App.css [hunk 1]
```diff
@@ -1,38 +1,18 @@
...
-    transform: rotate(360deg);
-  }
+  text-align: center;
+  overflow-y: hidden;
+  overflow-x: hidden;
+  margin: 3% 3%;
+  background: #FBF3EE;
+  height: 80vh;
+  border-radius: 50px;
+  padding-left: 3%;
```
Comment (krdurrett) [inline src/styles/App.css:13]: These all look fabulous!

File: src/styles/App.css [hunk 2]
```diff
@@ -1,38 +1,18 @@
...
+  padding-right: 3%;
+  padding-bottom: 3%;
+  box-shadow: 8px 9px 1px #595045,
```
Comment (cagallo) [inline src/styles/App.css:16]: Love the box shadow, such a nice effect!

File: src/styles/BookCard.css [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+.book-side-panel {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  margin-left: 8%;
+  text-align: left;
+  width: 100%;
+}
+
+.book-cover {
```
Comment (krdurrett) [inline src/styles/BookCard.css:26]: Love the book cards

File: src/styles/BookContainer.css [hunk 4]
```diff
@@ -0,0 +1,25 @@
+.book-container {
```
Comment (cagallo) [inline src/styles/BookContainer.css:1]: Smart to change to grid, have had issues with making boxes/cards with flexbox!

File: src/styles/BookContainer.css [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  grid-row-gap: 12%;
+  grid-column-gap: 4.9%;
+  justify-content: center;
+  overflow-y: scroll;
+  height: 80%;
+  width: 100%;
+  padding-bottom: 2.6%;
+}
+
+/* .book-container {
```
Comment (krdurrett) [inline src/styles/BookContainer.css:14]: Keeping this for later

**Review by cagallo — APPROVED** [review]
This looks great!

File: src/styles/Nav.css [hunk 6]
```diff
@@ -0,0 +1,18 @@
+.navbar {
```
Comment (krdurrett) [inline src/styles/Nav.css:1]: The nav bar looks great.

File: src/styles/BookCard.css [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+.book-author {
+  font-size: small;
+}
+
+.learn-more-container {
+  width: 8em;
+  height: 2.5em;
+}
+
+.learn-more-btn {
```
Comment (Fordo29) [inline src/styles/BookCard.css:47]: So much styling - thank you for working so hard.. coverting a link into a button

**Review by krdurrett — APPROVED** [review]

**Pull request merged**